### PR TITLE
Changed docker-compose file.

### PR DESCRIPTION
**Pull request by VladimirNovgorodov**

### CODE REVIEW

**Comment by akolotov** [comment]
@akolotov @fvictorio please take a look at this PR

**Comment by pertsev** [comment]
@VladimirNovgorodov Please, add CPU and RAM limits for containers.

File: .dockerignore [hunk 1]
```diff
@@ -1,3 +1,11 @@
-node_modules
```
Comment (patitonar) [inline .dockerignore]: I think `node_modules` should be ignored. I see there was a similar conversation on this comment https://github.com/poanetwork/token-bridge/pull/109#discussion_r227427251

**Comment by akolotov** [comment]
@VladimirNovgorodov could you share a forecast when the comments raised in this PR are addressed?
We need to provide responses on the issues found during the security audit on this week. Is it possible?

@phahulin, fyi

**Comment by VladimirNovgorodov** [comment]
Yes.  I'm working on it.

**Comment by VladimirNovgorodov** [comment]
> @VladimirNovgorodov Please, add CPU and RAM limits for containers.
Docker-compose version 3 that we use only supports these limits in swarm mode. https://forums.docker.com/t/memory-limit-in-version-3-compose-files-without-using-swarm/37864. 
In this case, I need to know if we can use Docker-compose file version 2 or maybe some other options.

**Comment by patitonar** [comment]
@VladimirNovgorodov should we close this PR in favor of #116 ?

**Comment by phahulin** [comment]
Yes

**Pull request closed without merging**